docs/go/errors.md

```diff
@@ -26,21 +26,21 @@ Handlers should return coded errors; if they don't, Connect will use the
 ```go
 func (s *greetServer) Greet(
   ctx context.Context,
-  req *connect.Request[greetv1.GreetRequest],
-) (*connect.Response[greetv1.GreetResponse], error) {
+  req *greetv1.GreetRequest,
+) (*greetv1.GreetResponse, error) {
   if err := ctx.Err(); err != nil {
     return nil, err // automatically coded correctly
   }
-  if err := validateGreetRequest(req.Msg); err != nil {
+  if err := validateGreetRequest(req); err != nil {
```

Comment:
As a result, the actual message is equal to the request. No need for a separate call to `req.Msg`.